### ISSUE TITLE
[FP]: (regression of #6695) CVE-2022-21704 still gets flagged on a Node.js dependency, 

### ISSUE THREAD

**Issue by volkert-fastned** [dependency-check/DependencyCheck#6869]
### Package URl

pkg:maven/org.apache.logging.log4j/log4j-slf4j2-impl@2.23.1

### CPE

`cpe:2.3:a:log4js_project:log4js:2.23.1:*:*:*:*:*:*:*`

### CVE

CVE-2022-21704

### ODC Integration

{"label"=>"Gradle Plugin"}

### ODC Version

10.0.3

### Description

As I stated in [this comment](https://github.com/jeremylong/DependencyCheck/issues/6695#issuecomment-2135247712), the previous regression didn't work.

In [this other comment](https://github.com/jeremylong/DependencyCheck/commit/b2052fb3f7e2bb86676303a3475025e4ce3077aa#r142470093), I shared required change to make it work.

TL;DR: the CPE section in the suppression has to be changed as follows:

```xml
<cpe>cpe:/a:log4js_project:log4js</cpe>
```

**Comment by volkert-fastned** [comment]
This yet another example of a FP as result of a platform/language mismatch. @OrangeDog FYI

**Comment by aikebah** [comment]
The original suppression from #6695 has been corrected and FP suppression automation has published the updated hosted suppressions file
